buildscripts/kokoro/android-interop.sh

```diff
@@ -15,6 +15,9 @@ export LDFLAGS=-L/tmp/protobuf/lib
 export CXXFLAGS=-I/tmp/protobuf/include
 export LD_LIBRARY_PATH=/tmp/protobuf/lib
 export OS_NAME=$(uname)
+export ANDROID_SDK_HOME=/opt/android-sdk/current
+
+echo y | ${ANDROID_SDK_HOME}/tools/bin/sdkmanager "build-tools;28.0.3"
```

Comment:
Hmm... this will actually install this particular version. I sort of question if it would be better to drop the right MD5s into licenses/android-sdk-license

Reply:
linux-artifacts is also failing. The same issue. http://g/kokoro-users/_P4EAOWi-ko/PRFQvowPDgAJ

Reply:
>I sort of question if it would be better to drop the right MD5s into licenses/android-sdk-license

I don't understand how to do that/ how that works.

Reply:
Hmm... it seems the license handling is done differently these days. Previously after you accepted a licenses you could get any of the binaries with that license. But that doesn't seem to work any more. So this seems fine.